### PR TITLE
CB-13068: Updated README to point to transition blog post and show cu…

### PR DESCRIPTION
**Pull request by maverickmishra**
### Platforms affected
iOS, Android and Windows

### What does this PR do?
This PR adds a deprecation notice and points to a transition blog post

### CODE REVIEW

**Comment by filmaj** [comment]
Isn't this at odds with @macdonst's PR open in #53 ?

**Comment by maverickmishra** [comment]
Yes, but I thought that issue was superseded by CB-13068.

**Comment by filmaj** [comment]
Yeah but it's basically the same stuff, no? I think we should combine what you two have done. I like that you link to the transition blog post. I like that Simon also updated the plugin and package json files. Thoughts @macdonst ?

**Comment by filmaj** [comment]
I'd also suggest we add an [`<info>` tag to the plugin.xml](https://cordova.apache.org/docs/en/latest/plugin_ref/spec.html#info) where we can be more noisy about the deprecation status of the plugin (as it will be printed out during plugin install).

File: README.md [hunk 1]
```diff
@@ -27,6 +27,18 @@ description: Access accelerometer data.
 
 # cordova-plugin-device-motion
 
+# Deprecation Notice
+
+With the [W3C Device Motion and Orientation API](https://www.w3.org/TR/2016/CR-orientation-event-20160818/) now
+being supported on iOS, Android and Windows devices, this plugin is not needed any more. Migrating from this
+plugin to the [W3C Device Motion and Orientation API](https://www.w3.org/TR/2016/CR-orientation-event-20160818/)
+is explained in this [PhoneGap blog post](https://blog.phonegap.com/migrating-from-the-cordova-device-motion-plugin-ddd8176632ed).
```
Comment (janpio) [inline README.md]: Why a Phonegap blogpost on a Cordova plugin deprecation?

Shouldn't there at least be a Cordova "we deprecate those plugins" blog post linked here and then maybe additionally a Phonegap blogpost with "this is how you do it now" (although I would of course prefer those at Cordova as well, much less confusing for users re Cordova/Phonegap).

Comment (stevengill) [reply README.md]: The plugins release blog post will talk about the deprecation.

**Comment by stevengill** [comment]
Merged this in, Added `info` tag. 

@maverickmishra can you close this issue?

**Pull request closed without merging**